docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -75,6 +76,54 @@ i.e. you cannot bind a horizontal pod autoscaler to a replication controller and
 The reason this doesn't work is that when rolling update creates a new replication controller,
 the horizontal pod autoscaler will not be bound to the new replication controller.
 
+## Support for custom metrics
+
+Kubernetes 1.2 brings an alpha support for scaling based on application-specific metrics like QPS(Query per second) or average request latency.
```

Comment:
Kubernetes {{ page.version }} adds alpha support for scaling based on application-specific metrics like QPS (queries per second) or average request latency.